### ISSUE TITLE
Wallet Unlocking Issue During lncli Setup

### ISSUE THREAD

**Issue by ghost** [MrRGnome/PayMeBTC-client#5]
During the initial setup of lncli with the HTML interface, the process incorrectly attempts to enforce the unlocking of a wallet that is already unlocked. This unexpected behavior can lead to confusion and potentially hinder the setup process.

**Comment by MrRGnome** [comment]
Can you confirm, is the specific text you're seeing "Your wallet is currently locked. Please unlock it. "?

**Comment by ghost** [comment]
Yes, that is correct. It only occurred once during setup though, so this might be hard to debug, and maybe a very low priority issue to fix.

**Comment by MrRGnome** [comment]
That's text returned from the LOCKED state attribute returned by lndapi/v1/state. If that text is there, it's because the LND api returned the locked state. Is it possible LND is lying during start ups?

**Comment by ghost** [comment]
I attempted to reproduce the error and I am fairly certain it lies in how the code bridges a connection with lnd. The connection for example will be bridged before any of the other parameters are set. This leads me to believe that there are undefined or at the very least states that shouldn't be reached.
